contrib/test_timm.py

```diff
@@ -0,0 +1,10 @@
+def test_push_to_hub(repo_name: str, cleanup_repo: None) -> None:
```

Comment:
I would also add a test that shows that you can download models

Reply:
Yes! Will do it once we validate the architecture (I have added a TODO in PR description for it).